includes/API.php

```diff
@@ -148,6 +148,47 @@ public function get_catalog( $catalog_id ) {
 	}
 
 
+	/**
+	 * Gets a user object from Facebook.
+	 *
+	 * @since 2.0.0-dev.1
+	 *
+	 * @param string $user_id user ID. Defaults to the currently authenticated user
+	 * @return API\User\Response
+	 * @throws Framework\SV_WC_API_Exception
+	 */
+	public function get_user( $user_id = '' ) {
+
+		$request = new API\User\Request( $user_id );
+
+		$this->set_response_handler( API\User\Response::class );
+
+		return $this->perform_request( $request );
+	}
+
+
+	/**
+	 * Delete's a user's API permission.
+	 *
+	 * This is their form of "revoke".
```

Comment:
should we then change the name of this method into `revoke_user_permission()` to make it more clear?